aten/src/ATen/native/native_functions.yaml

```diff
@@ -14068,12 +14068,13 @@
     CUDA: _flash_attention_backward
 
 # Returns ouput, logsumexp if compute_logsumexp
-- func: _efficient_attention_forward(Tensor query, Tensor key, Tensor value, Tensor? cu_seqlens_q, Tensor? cu_seqlens_k, int? max_seqlen_q, bool compute_log_sumexp=False, bool causal=False, *, float? scale=None) -> (Tensor, Tensor)
```

Comment:
Key assumption: Since these functions are "hidden" to torchscript behind the top level `torch._C._nn._scaled_dot_product_attention()` I am safe to change these signatures willy nilly

Reply:
So far this has proven correct